src/js/UserManager.js

```diff
@@ -0,0 +1,18 @@
+import { createUserManager } from 'redux-oidc'
+
+const userManager = createUserManager({
+  authority: 'https://192.168.99.100:8443/auth/realms/hmda',
```

Comment:
How is the value passed in, then? Environment variable, some discovery endpoint, etc?